app/lib/katello/concerns/base_template_scope_extensions.rb

```diff
@@ -59,6 +59,14 @@ def host_content_facet(host)
         host.content_facet
       end
 
+      apipie :method, "Returns the value of Setting['deb_enable_structured_apt']" do
+        desc 'Structured APT is a feature for the deb content type that may be disabled'
+        returns one_of: [true, false]
+      end
+      def deb_using_structured_apt?
+        Setting['deb_enable_structured_apt']
+      end
```

Comment:
Is this really needed? I'd think you can just use the existing Settings API [here](https://github.com/theforeman/foreman/blob/develop/app/controllers/api/v2/settings_controller.rb), which enables `hammer host settings info --name deb_enable_structured_apt`

Reply:
I will remove this again. We use it on our downstream branch so that a provisioning template we deliver can check the setting. However, the template in question does not exist in upstream Katello, and anyone who writes their own template can just adjust it after enabling/disabling "structured APT", so I don't think there is an upstream use case for this.

Reply:
Dropped.